Source/Extensions/Blazorise.DataGrid/_DataGridCell.razor.cs

```diff
@@ -34,10 +30,28 @@ protected Action<ValidatorEventArgs> Validator
         protected string ValidationPattern
             => string.IsNullOrWhiteSpace( Column.ValidationPattern ) ? null : Column.ValidationPattern;
 
+        protected Type ValidationHandlerType
```

Comment:
Is this clear to the user? Does the understand that the 3 existing methods of validation are mutually exclusive? 
Can there be a scenario where he has a validator or validation pattern but simply wants to use the data annotations? Or any other combination? 
Which could warrant having a `ValidationMode `enum.

And Is there a scenario where we want to run every single one of them?

Reply:
Most people want to use data annotation so I made it the first rule. Every other will override it just as in the regular validation. We can make it more clear in the documentation.

Reply:
Personally I could see myself using the data annotations for simple stuff,and then add a validator or validation pattern to do some more complex logic all on the same column. So I do not need to redo the simple validations according to the mode I'm using.
But, to be fair I think data annotations does have its own custom validation and pattern too.

Reply:
Updated documentation.